pkgs/os-specific/darwin/dockutil/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchurl, libarchive, p7zip }:
+
+stdenv.mkDerivation rec {
+  pname = "dockutil";
+  version = "2.0.5";
+
+  src = fetchurl {
+    url = "https://github.com/kcrawford/dockutil/releases/download/${version}/dockutil-${version}.pkg";
```

Comment:
Why download pkg file when we can get the same scripts with fetchFromGithub?

Reply:
I took the release assets to have a specific version pinned. Since these are my first packages for nixpkgs I'm not sure what else is the common handling for such cases.

Reply:
Each github release corresponds to specific tag, which you can specify in `rev` argument to `fetchFromGithub`.